chain/node.go

```diff
@@ -52,6 +53,13 @@ func (n *NodeClient) BlockByHash(ctx context.Context, hash string) (*gateway.Blo
 	return b, httpGet(ctx, url, b)
 }
 
+// Account gets account detail by account address
+func (n *NodeClient) Account(ctx context.Context, address string) (*gateway.AccountResp, error) {
```

Comment:
shall we perform a sanity check for the address before sending out req? This applies to other APIs